src/content/docs/en/welcome.mdx

```diff
@@ -79,18 +90,18 @@ Fyra Labs (the company behind Ultramarine) does not collect any data that can be
 
 ### Is Ultramarine semi-rolling?
 
-**No**. Ultramarine and Fedora are not semi-rolling or rolling. Instead, they use a stable release scheme.
-Updates with breaking changes are rolled out only for every stable release version. Minor updates / point updates are guaranteed to be backwards-compatible, i.e. system components in Ultramarine are updated if and only if they do not break compatibility.
+No. Ultramarine and Fedora are not semi-rolling or rolling. Instead, they use a stable release scheme.
+Updates with breaking changes are rolled out for major releases. Minor updates/point updates are backwards-compatible. Put simply, system components in Ultramarine are updated if and only if they do not break compatibility.
 
-However, Ultramarine Linux includes Terra, RPM Fusion and Flathub.
+However, Ultramarine Linux includes Terra, RPMFusion and Flathub.

```

Comment:
I said RPM Fusion is called RPM Fusion, please go to the official website.

Reply:
oh shit, it is